install-dev/upgrade/upgrade.php

```diff
@@ -288,6 +289,31 @@
 
 Context::getContext()->smarty = $smarty;
 
+// Disable the old incompatible modules
+$config = Symfony\Component\Yaml\Yaml::parse(__DIR__.'/../../app/config/config.yml');
+$baseUrl = $config['csa_guzzle']['clients']['addons_api']['config']['base_url'];
```

Comment:
I guess this won't be the only place where this piece of code will be needed.

Don't you think we should move it in the ModuleManager in case of the Symfony fernel is not instanciated ?